sys/shell/commands/sc_gnrc_rpl.c

```diff
@@ -313,11 +315,9 @@ int _gnrc_rpl_dodag_show(void)
                 gnrc_rpl_instances[i].mop, gnrc_rpl_instances[i].of->ocp,
                 gnrc_rpl_instances[i].min_hop_rank_inc, gnrc_rpl_instances[i].max_rank_inc);
         LL_FOREACH(gnrc_rpl_instances[i].dodags, dodag) {
-            tc = timex_sub(dodag->trickle.msg_callback_timer.absolute, now);
-            ti = timex_sub(dodag->trickle.msg_interval_timer.absolute, now);
+            tc = (dodag->trickle.msg_callback_timer.target - xnow);
+            ti = (dodag->trickle.msg_interval_timer.target - xnow);
             cleanup = timex_sub(dodag->cleanup_timer.absolute, now);
```

Comment:
nope, did not forget. This change is handled in #3778, because it has nothing to do with trickle